### DISCUSSION TITLE
Intended use of save-symbol

### DISCUSSION THREAD

**Discussion by DanielAjoy** [h3rald/min#181] [General]
Reading the reference I discovered the save-symbol operator, but I didn't find a mention of it's intended use. Is it just way to save frequently used lambdas and other constants from session to session? Like a way to create a personal dictionary of words?

**Comment by DanielAjoy** [comment]
To have those symbols readily available next session, I added this line to the **.minrc** file:

`saved-symbols (dup load-symbol ROOT publish) foreach`

Again, I'm not sure if that was the intended of the **.min_symbols** file.

**Reply by h3rald** [reply]
Yup that's correct. Basically whenever you call `save-symbol` the definition is serialized to JSON and stored to the .min_symbols file.

You can use `load-symbol` to load it or, like you did, add some code to your .minrc file to auto-load all saved symbols.
